aws/adapter.go

```diff
@@ -248,6 +248,17 @@ func NewAdapter(clusterID, newControllerID, vpcID string, debug, disableInstrume
 	return
 }
 
+// UpdateManifest generates a manifest again based on cluster and VPC information.
+// This method is useful when using custom AWS clients.
+func (a *Adapter) UpdateManifest(clusterID, vpcID string) (*Adapter, error) {
```

Comment:
The need for this method suggest to me we are not doing this in the best way as we have both `NewAdapter` and `UpdateManifest` calling `buildManifest`. It looks like an opportunity to restructure this a bit so it's not needed. Doesn't have to be done in this PR, but it just came to my attention when looking at it.

Reply:
Yeah, I agree, I didn't refactor this at the time because I thought it was too much for this task. I created a task to do it in a next step: https://github.bus.zalan.do/pg9/issues/issues/412